djoser/locale/fr/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,139 @@
+"POT-Creation-Date: 2019-05-14 18:40+0200\n"
+"PO-Revision-Date: 2019-05-14 19:03+0200\n"
+"Last-Translator: Julie Rymer <chadys@hotmail.fr>\n"
+"Language-Team: French\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#: constants.py:4
+msgid "Unable to log in with provided credentials."
+msgstr "Impossible de se connecter avec les identifiants fournis."
+
+#: constants.py:5
+msgid "User account is disabled."
+msgstr "Ce compte utilisateur est désactivé."
+
+#: constants.py:6
+msgid "Invalid token for given user."
+msgstr "Le jeton d'authentification est invalide pour cet utilisateur."
+
+#: constants.py:7
+msgid "Invalid user id or user doesn't exist."
+msgstr "L'id de l'utilisateur est invalide ou cet utilisateur n'existe pas."
+
+#: constants.py:8
+msgid "Stale token for given user."
+msgstr "Le jeton pour cet utilisateur est expiré."
+
+#: constants.py:9
+msgid "The two password fields didn't match."
+msgstr "Le contenu des deux champs mot de passe est différent."
+
+#: constants.py:10
+#, python-brace-format
+msgid "The two {0} fields didn't match."
+msgstr "Le contenu des deux champs {0} est différent."
+
+#: constants.py:11
+msgid "Invalid password."
+msgstr "Mot de passe invalide."
+
+#: constants.py:12
+msgid "User with given email does not exist."
+msgstr "Cette adresse email ne correspond à aucun utilisateur enregistré."
+
+#: constants.py:13
+msgid "Unable to create account."
+msgstr "Création de compte impossible."
+
+#: constants.py:15
+msgid ""
+"User model does not contain specified email field. Please see http://djoser."
+"readthedocs.io/en/latest/settings.html#USER_EMAIL_FIELD_NAME for more "
+"details."
+msgstr ""
+"Le modèle User ne contient pas le champ email indiqué. Veuillez consulter "
+"http://djoser.readthedocs.io/en/latest/settings.html#USER_EMAIL_FIELD_NAME "
+"pour plus de détails."
+
+#: templates/email/activation.html:4
+#, python-format
+msgid "Account activation on %(site_name)s"
+msgstr "Activation du compte sur %(site_name)s"
+
+#: templates/email/activation.html:8 templates/email/activation.html:19
+#, python-format
+msgid ""
+"You're receiving this email because you need to finish activation process on "
+"%(site_name)s."
+msgstr ""
+"Vous recevez cet email car vous devez finir le processus d'activation de votre "
+"compte sur %(site_name)s."
+
+#: templates/email/activation.html:10 templates/email/activation.html:22
+msgid "Please go to the following page to activate account:"
+msgstr "Veuillez cliquer sur le lien suivant pour activer votre compte :"
+
+#: templates/email/activation.html:13 templates/email/activation.html:25
+#: templates/email/confirmation.html:10 templates/email/confirmation.html:18
+#: templates/email/password_reset.html:14
+#: templates/email/password_reset.html:26
+msgid "Thanks for using our site!"
+msgstr "Merci d'avoir utilisé notre site !"
```

Comment:
Is the space before exclamation mark normal in French?

Reply:
Yes, in French most punctuation needs a space before